etc/di.xml

```diff
@@ -2,4 +2,17 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <preference for="Klaviyo\Reclaim\Api\ReclaimInterface"
                 type="Klaviyo\Reclaim\Model\Reclaim" />
+    <type name="Klaviyo\Reclaim\Logger\Handler">
```

Comment:
any way to pull these string literals out into a constant? at least for the ones that are repeated?

Reply:
eh, it's okay to leave it imo, it's a magento standard, we shouldn't really deviate from it